Cue/app/common/CueDrawerLayout.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+var React = require('React');
+var DrawerLayoutAndroid = require('DrawerLayoutAndroid');
+
+class CueDrawerLayout extends React.Component {
+  _drawer: ?DrawerLayoutAndroid;
+
+  constructor(props: any, context: any) {
+    super(props, context);
+
+    (this: any).openDrawer = this.openDrawer.bind(this);
+    (this: any).closeDrawer = this.closeDrawer.bind(this);
+    (this: any).onDrawerOpen = this.onDrawerOpen.bind(this);
+    (this: any).onDrawerClose = this.onDrawerClose.bind(this);
+    (this: any).handleBackButton = this.handleBackButton.bind(this);
+  }
+
+  render() {
+    const {drawerPosition, ...props} = this.props;
+    const {Right, Left} = DrawerLayoutAndroid.positions;
```

Comment:
Strangely, I'm not able to get the app running on iOS until I comment this line (and line 26) out, otherwise I get the error:
```
undefined is not an object (evaluating '_DrawerLayoutAndroid$.Right')
```

My Google-fu isn't turning anything up, and in the interest of saving time, since I don't think we'll ever need to actually have the menu open from the right side, I suggest that we (temporarily) remove this config option until we can come to a solution?